### PR TITLE
Nodes page

### PR DESCRIPTION
**Pull request by Flaburgan**
Each node now has its specific page accessible by clicking on its name from the list of nodes. All the info about the pod and its charts can be found on its page. This PR solves the problem of the charts by node freezing the browser with the current version of the-federation.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -5,6 +5,31 @@ var utils = {},
     mailer = require("./mailer"),
     config = require('./config');
 
+utils.formatPodInfo = function(pod) {
```
Comment (jaywink) [inline src/utils.js]: I'd prefer any new code always use the term `node` over `pod` - as this site is not only related to diaspora. Mind going through the whole PR and fixing those?

**Comment by jaywink** [comment]
Also, rebase of course :)

**Comment by Flaburgan** [comment]
I found (and fixed) the regression on table sorting by node name. This is ready to be merged.

**Comment by jaywink** [comment]
Ok let's merge, thanks!

**Pull request merged**